script.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import pprint
+import config
+
+def main():
+    token = generateToken(config.client_id, config.client_secret)
+    if token:
+        response = requestVehicles(token)
+        printInfo(response)
+    return 0
+
+def generateToken(client_id, client_secret):
+    tokenResponse = requests.post("https://api.oregonstate.edu/oauth2/token", data={
```

Comment:
`https://api.oregonstate.edu/` is actually used twice, can you store it in a variable and make it reusable?